bundles/AdminBundle/Resources/public/js/pimcore/object/tags/input.js

```diff
@@ -52,10 +52,13 @@ pimcore.object.tags.input = Class.create(pimcore.object.tags.abstract, {
         var input = {
             fieldLabel: this.fieldConfig.title,
             name: this.fieldConfig.name,
-            componentCls: "object_field",
             labelWidth: 100
         };
 
+        if (!this.fieldConfig.showCharCount) {
```

Comment:
what is the intention of this switch?

Reply:
I think the css class should only be once per field and at the outer component. If showCharCount is activated, the text field get a container with this class.

Reply:
That's actual a good question. I would agree with you.
However, if there is a charCount then it is not added at all?

Reply:
If there is a charCount it will added to the Panel:
https://github.com/pimcore/pimcore/pull/6873/files/7186d47bf7f5d5f6eab95d03cd738db3ed6eb42a#diff-05180e356c10a77013221780b2e13ebaR106

Reply:
aah, it is getting late :-) then everything should be fine ...